### ISSUE TITLE
why 'Train base only' achieves the novel mAP 9.0?

### ISSUE THREAD

**Issue by squirrel16** [ucbdrive/few-shot-object-detection#12]
Thanks for your good work! 
But there is a weird phenomenon in Table2, the _[Train base only]_ method achieves 9.0 nAP. As far as I understand, there is not any labeled novel instance while using this method. If any, maybe you ignore its label and treat it as background. Thus the network could not learn any class-specific information of a certain novel class, and the weights of classification and regression head of novel classes are just random initializations, why it can achieve nAP 9.0?
I'm looking forward to your answer.

**Comment by thomasehuang** [comment]
Hello, thank you for noticing this issue.

We originally assumed the 9.0 nAP was due to randomness, but it is actually a mistake on our end and we reported the wrong number. Instead of random initialization, we used a model where the novel weights were fine-tuned on the novel data first and then combined together with the base weights.

We re-evaluated the base model and it was able to achieve bAP50: 77.9 and nAP50: 1.0. We will update the paper with these new values when we have time.

Hope that helps!
